app/presenters/hyrax/admin/users_presenter.rb

```diff
@@ -0,0 +1,33 @@
+module Hyrax
+  module Admin
+    class UsersPresenter
+      def initialize
+        @users = users
```

Comment:
I chose to build the @users variable upon initialization, because the count of users method is accessed before the users method in the view, and they both share the same method to load the data.